### ISSUE TITLE
Unify the markup for buttons with icons

### ISSUE THREAD

**Issue by JulianKniephoff** [opencast/annotation-tool#149]
Sometimes this is an `<i>` within a `<button>`, sometimes there is no `<button>`. This subtly affects browser's treatment of these; see for example #140.

Related to this, there is the issue of accessibility with these; there should really be some kind of label for each of these, that can be understood properly by assistive technology. The markup for these should be consistent as well, of course. Tangentially related: #147.
